### DISCUSSION TITLE
Set cursor position automatically

### DISCUSSION THREAD

**Discussion by mttjhn** [SilentVoid13/Templater#490] [Help] [answered]
I'm struggling to figure out how to get the cursor position to move to my first `<% tp.file.cursor() %>` location automatically. Somehow I found on the Obsidian forums that you can press the hotkey (OPT-TAB) to get it to move to the next location, but I'd rather not have to remember that keystroke. Is there a way to get the template to jump to that location once everything has been processed? Or is there a hidden setting I'm missing? Thanks in advance for the help!

**Comment by chenxin-yan** [comment]
Turn on Trigger Templater on new file creation in the setting

**Comment by mttjhn** [comment, marked as answer]
Thanks for suggesting I look at the settings @chenxinYan2004! Turns out there's a setting for this:
![TemplaterCursor](https://user-images.githubusercontent.com/1553972/147977653-56beef7e-b2fe-4f45-96a9-2c56a9a1b74e.png)

**Reply by chenxin-yan** [reply]
For me, only turning on Automatic jump to cursor does not solve the issue, I have to turn on Trigger Templater on new file creation in order for the tp.file.cursor to work without hotkey.
